src/main/java/ch/naviqore/service/ArrivalTime.java

```diff
@@ -0,0 +1,9 @@
+package ch.naviqore.service;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDate;
+
+public interface ArrivalTime {
+    @NotNull LocalDate getArrivalTime();
```

Comment:
Should we introduce / use this annotation? It gets quite verbose...

Until now we just marked @Nullable fields and assumed that fields without annotations are never null.